docs/metadata/configure-audit-logging.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Configure Audit Logging
+id: auditLogging
+description: Prophecy Audit logs available to be exported in S3
+sidebar_position: 6
+tags:
+  - Audit Logs
+---
+
+Prophecy provides access to audit logs of activities performed by Prophecy users, allowing your enterprise to monitor detailed usage patterns.
+User can configure a S3 bucket to sync these events from Prophecy to their environment.
+
+## Configure S3 bucket for logs
```

Comment:
We're missing a little bit of an introduction here. It should answer questions like: Is this for Public SaaS only or all environments? High-level process description. Is it manual?

Reply:
This applies for Public SaaS only .. Right now its manual from both customer/Our side .. We will add these details